### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Resolves:** #97 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.
- Update font-weight and sizes for respective text components

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by fajar-apri-alaska** [comment]
The  `npm ci` is failing because some auro components have conflicting peer dependency for `@aurodesignsystem/webcorestylesheets` package. Should I hold of this until those components are updated?

**Comment by blackfalcon** [comment]
This PR is `BLOCKED` until this PR is approved and shipped. 

- https://github.com/AlaskaAirlines/auro-badge/pull/85

**Comment by blackfalcon** [comment]
This branch needs to be repointed from `main` to a new RC branch.

**Comment by fajar-apri-alaska** [comment]
This PR also `BLOCKED` by:
- https://github.com/AlaskaAirlines/auro-flightline/pull/59

until it is approved, merged, and released.

**Comment by blackfalcon** [comment]
> This PR also `BLOCKED` by:
> 
> * [Update CSS token per new design system theming capability auro-flightline#59](https://github.com/AlaskaAirlines/auro-flightline/pull/59)
> 
> until it is approved, merged, and released.

Shipped. @fajar-apri-alaska please update repo and update status of issue to 'design review needed' to ensure that issue #102 has also been addressed.

File: src/style-flight-main.scss
Comment (blackfalcon) [inline src/style-flight-main.scss:46]: Auro doesn't support a `600` weight? Only `100`, `300` and `500`. See https://github.com/AlaskaAirlines/WebCoreStyleSheets/blob/master/src/_fonts.scss

Where is the `600` spec coming from?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

File: src/style-flight-main.scss
Comment (fajar-apri-alaska) [reply src/style-flight-main.scss:46]: I'm not sure on this one, maybe @jordanjones243 could help answer to this?

Comment (jordanjones243) [reply src/style-flight-main.scss:46]: Font-weight 600 is coming as a request from the flight search team as they said the current UI does not match what they have in their design. Although, there is no blueprint for this component.

Comment (jason-capsule42) [reply src/style-flight-main.scss:46]: @jordanjones243 is there a ticket for this font weight request? That needs to be done separate from this body of work.

@fajar-apri-alaska Since we don't have a font-weight 600 currently let's not include this part of the change and we will handle it in the ticket above.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
See comments about font-weight

File: src/style-flight-main.scss
Comment (jordanjones243) [reply src/style-flight-main.scss:46]: @leeejune is currently developing a blueprint for auro-flight. We can put this work on hold and remove it from this PR until that blueprint is finished.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**
